weddineo-frontend/apps/weddineo-frontend/src/app/app.module.ts

```diff
@@ -26,6 +30,14 @@ import { TokenInterceptor } from './shared/interceptor/token.interceptor';
         },
       }
     ),
+    TranslateModule.forRoot({
+      defaultLanguage: 'en_GB',
```

Comment:
I think that default languege should be polish. 
@PrzemyslawZnojek can you confirm?